src/roving-tabindex-mixin.ts

```diff
@@ -0,0 +1,60 @@
+import { LitElement } from 'lit-element';
+import { KeyboardDirectionInterface, getAvailableIndex, isFocusable, $focus } from './keyboard-direction-mixin';
+import { $itemsChanged, SlottedItemsInterface } from './slotted-items-mixin';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T = object> = new (...args: any[]) => T;
+
+const $setFocusable = Symbol('setFocusable');
+
+const $setTabindex = Symbol('setTabindex');
+
+export interface RovingTabindexInterface {
+  [$setFocusable](idx: number): void;
+  [$setTabindex](item: HTMLElement): void;
+}
+
+export type RovingTabindexConstructor = Constructor<RovingTabindexInterface>;
+
+export const RovingTabindexMixin = <
+  T extends Constructor<KeyboardDirectionInterface & SlottedItemsInterface & LitElement>
+>(
+  base: T
+): T & RovingTabindexConstructor => {
+  class RovingTabindex extends base {
```

Comment:
The name of the mixin and all variables and references to "tabindex" should have a capital "i" as in the `HTMLElement` property name `tabIndex`. It looks weird to me that now "index" is all lowercase in the middle of many names.

I think it's fine to keep the file names as `*-tabindex-*` (though I wouldn't mind if those were renamed too like `roving-tab-index-mixin.ts`). But at least I would prefer the variables/classes to look more consistent.

I would rename at least:
- `Symbol('setTabindex')` -> `Symbol('setTabIndex')`
- `$setTabindex` -> `$setTabIndex`
- `RovingTabindexInterface` -> `RovingTabIndexInterface`
- `RovingTabindexConstructor ` -> `RovingTabIndexConstructor `
- `RovingTabindexMixin` -> `RovingTabIndexMixin`
- `RovingTabindex` -> `RovingTabIndex`

Reply:
Thanks, done.